python/src/llvm.cc

```diff
@@ -302,6 +303,21 @@ void init_triton_llvm(py::module &&m) {
         PassBuilder pb(nullptr /*targetMachine*/, tuningOptions, std::nullopt,
                        instrCbPtr);
 
+        std::string pluginFile =
+            mlir::triton::tools::getStrEnv("AMDGCN_INSTRUMENTATION_LIB");
```

Comment:
Does this need to be AMD specific? I think this is just general LLVM pass plugin mechanism? What about naming it as `LLVM_PASSPLUGIN_PATH`?

Reply:
No, you're correct it's just using the standard LLVM pass plug-in infrastructure. Updated ENV to be architecture agnostic.